cadence/contracts/bridge/FlowEVMBridgeHandlers.cdc

```diff
@@ -191,14 +192,229 @@ access(all) contract FlowEVMBridgeHandlers {
         }
     }
 
+    /// Facilitates moving Flow between Cadence and EVM as WFLOW. Since WFLOW is an artifact of the EVM ecosystem, 
+    /// wrapping the native token as an ERC20, it does not have a place in Cadence's fungible token ecosystem.
+    /// Given the native interface on EVM.CadenceOwnedAccount and EVM.EVMAddress to move FLOW between Cadence and EVM,
+    /// this handler treats requests to bridge FLOW as WFLOW as a special case.
+    ///
+    access(all) resource WFLOWTokenHandler : FlowEVMBridgeHandlerInterfaces.TokenHandler {
+        /// Flag determining if request handling is enabled
+        access(self) var enabled: Bool
+        /// The Cadence Type this handler fulfills requests for. This field is optional in the event the Cadence token
+        /// contract address is not yet known but the EVM Address must still be filtered via Handler to prevent the
+        /// contract from being onboarded otherwise.
+        access(self) var targetType: Type
+        /// The EVM contract address this handler fulfills requests for.
+        access(self) var targetEVMAddress: EVM.EVMAddress
+
+        init(wflowEVMAddress: EVM.EVMAddress) {
+            self.enabled = false
+            self.targetType = Type<@FlowToken.Vault>()
+            self.targetEVMAddress = wflowEVMAddress
+        }
+
+        /// Returns whether the Handler is enabled
+        access(all) view fun isEnabled(): Bool {
+            return self.enabled
+        }
+        /// Returns the Cadence type handled by the Handler, nil if not set
+        access(all) view fun getTargetType(): Type? {
+            return self.targetType
+        }
+        /// Returns the EVM address handled by the Handler, nil if not set
```

Comment:
cannot be nil

Reply:
The function signature here is a manifestation of `FlowEVMBridgeHandlerInterfaces.HandlerInfo` interface conformance.